Projects/My_website/myWebsite/blog/views.py

```diff
@@ -26,16 +26,57 @@ class userBlog(APIView):
     authentication_classes = [TokenAuthentication]
     permission_classes = [IsAuthenticated]
 
+    def get_item(self,slug):
+        try:
+            item = Post.objects.get(id=slug)
+            return item
+        except:
```

Comment:
@codal-uchoudhary ,
Pls explain how can `id=slug` condition be used to fetch the Post instance ?

Reply:
so , now i add id of post in the URL

Reply:
<img width="655" alt="image" src="https://github.com/codal-uchoudhary/python-training-2024/assets/156191290/7d508dba-e819-44fa-975e-a99390ecbada">